app/models/storage_location.rb

```diff
@@ -132,6 +132,11 @@ def self.import_csv(csv, organization)
   # @param loc [Integer] StorageLocation ID
   # @return [void]
   def self.import_inventory(filename, org, loc)
+    storage_location = StorageLocation.find(loc.to_i)
+    storage_location_exists = storage_location.size > 0
```

Comment:
We need this to work both with and without event sourcing. Can you use the method `empty_inventory?`instead to check this? That's works with both.